exercises/farewell-workflow/solution/Workflow/TranslateActivities.cs

```diff
@@ -0,0 +1,23 @@
+// Translate Activities
```

Comment:
Usually no need for leading comments like this in any of the files (but if you want proper .NET doc you _could_, though that's probably unnecessary too)